open_academy/wizards/wizard_session_views.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+    <record id="session_view_form_wizard" model="ir.ui.view">
```

Comment:
I asked for a change here but delete it because I made a mistake (I forgot that the name of your model is wizard_session), this should be like

```suggestion
    <record id="wizard_session_view_form" model="ir.ui.view">
```